src/components/Cart/CartItems.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import "./Cart.css";
+export const CartItems = () => {
+  const { cartState, cartDispatch } = useCart();
+  const { wishlistState, wishlistDispatch } = useWishlist();
+
+  return (
+    <>
+      <div className="cart-product-container">
+        {cartState.map((product) => {
+          return (
+            <div className="cart-card" key={product._id}>
+              <div className="cart-img-container">
+                <img
+                  src={product.img}
+                  alt={product.title}
+                  className="cart-img"
+                />
+                <i
+                  className="fas fa-trash-alt delete"
+                  onClick={() =>
+                    cartDispatch({
+                      type: "REMOVE_FROM_CART",
+                      payload: product,
+                    })
+                  }
+                ></i>
+              </div>
+
+              <div className="card-content-hrz">
+                <h4 className="card-title-hrz">{product.title}</h4>
+                <p className="card-seller-hrz">by {product.author}</p>
+
+                <div className="quantity">
+                  <button
+                    className="increase-quantity"
+                    onClick={() =>
+                      cartDispatch({
+                        type: "INCREASE_QUANTITY",
+                        payload: product,
+                      })
+                    }
+                  >
+                    +
+                  </button>
+                  <span className="quantity-wt">{product.quantity}</span>
+                  <button
+                    className="decrease-quantity"
+                    onClick={() =>
+                      cartDispatch({
+                        type: "DECREASE_QUANTITY",
+                        payload: product,
+                      })
+                    }
+                    disabled={product.quantity === 1 ? true : false}
```

Comment:
- Here ternary is not required as we are performing a comparison here and  it will result out to be a boolean value itself so no need to return ```true ``` and ```false ```explicity

```suggestion
                    disabled={product.quantity === 1}
```